<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-10-24T17:41:15 IST" name="Suite" finished-at="2021-10-24T17:41:38 IST" duration-ms="23067">
    <groups>
    </groups>
    <test started-at="2021-10-24T17:41:15 IST" name="Test" finished-at="2021-10-24T17:41:38 IST" duration-ms="23067">
      <class name="TestRegression.TestCasesUIKW">
        <test-method is-config="true" signature="setUp()[pri:0, instance:TestRegression.TestCasesUIKW@2dc54ad4]" started-at="2021-10-24T17:41:15 IST" name="setUp" finished-at="2021-10-24T17:41:20 IST" duration-ms="5427" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="titleOfHomePage()[pri:0, instance:TestRegression.TestCasesUIKW@2dc54ad4]" started-at="2021-10-24T17:41:20 IST" name="titleOfHomePage" finished-at="2021-10-24T17:41:24 IST" duration-ms="3985" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleOfHomePage -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestRegression.TestCasesUIKW@2dc54ad4]" started-at="2021-10-24T17:41:24 IST" name="tearDown" finished-at="2021-10-24T17:41:25 IST" duration-ms="846" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestRegression.TestCasesUIKW -->
      <class name="uiTest.SignUPPageTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:uiTest.SignUPPageTests@7dc19a70]" started-at="2021-10-24T17:41:25 IST" name="setUp" finished-at="2021-10-24T17:41:27 IST" duration-ms="2141" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validateFirstNameField()[pri:0, instance:uiTest.SignUPPageTests@7dc19a70]" started-at="2021-10-24T17:41:27 IST" name="validateFirstNameField" finished-at="2021-10-24T17:41:35 IST" duration-ms="7535" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at uiTest.SignUPPageTests.validateFirstNameField(SignUPPageTests.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateFirstNameField -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:uiTest.SignUPPageTests@7dc19a70]" started-at="2021-10-24T17:41:37 IST" name="tearDown" finished-at="2021-10-24T17:41:38 IST" duration-ms="1011" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- uiTest.SignUPPageTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
